app/src/main/java/org/p2p/wallet/auth/interactor/MetadataFailed.kt

```diff
@@ -1,19 +1,19 @@
 package org.p2p.wallet.auth.interactor
 
-sealed class GetOnboardingMetadataFailed(
+sealed class MetadataFailed(
     override val message: String,
     override val cause: Throwable? = null
 ) : Throwable() {
-    class OnboardingMetadataRequestFailure(cause: Throwable) : GetOnboardingMetadataFailed(
+    class OnboardingMetadataRequestFailure(cause: Throwable) : MetadataFailed(
```

Comment:
maybe we could clarify and shorten these names like:
`MetadataFailed `->`UnknownError`
`MetadataNoAccount` -> `NoAccount`
`MetadataNoSeedPhrase` -> `NoSeedPhrase`?